src/txn/txn.c

```diff
@@ -157,12 +156,7 @@ __wt_txn_get_snapshot(WT_SESSION_IMPL *session)
 
 	/* Walk the array of concurrent transactions. */
 	WT_ORDERED_READ(session_cnt, conn->session_cnt);
-	ckpt_id = txn_global->checkpoint_id;
 	for (i = n = 0, s = txn_global->states; i < session_cnt; i++, s++) {
-		/* Skip the checkpoint transaction; it is never read from. */
```

Comment:
To be sure I understand.. Do we no longer need to skip the checkpoint transaction because it isn't setting a transaction ID in the regular ID table any more? It now only sets it in the ```txn_global->checkpoint_pinned``` field?